README.md

```diff
@@ -184,7 +184,7 @@ discontinued. For documentation on the older command options, refer to
 - `-h`, `--help`<br />
   Show commandline help, including a list of options, and sample use cases.
 
-- `-v`, `--version`<br />
+- `--version`<br />
```

Comment:
Ok. But keeping them in separate PRs helps to keep track of things when we need to look historically. Anyway, please move this out of this PR and into a separate one.